docs/presentation/Hierarchies/ChildNodeRule.md

```diff
@@ -1,34 +1,115 @@
+
+The most commonly used symbols are:
+
+- `ParentNode` to define which parent node this rule is creating child nodes for.
+
+  ```ts
+  [[include:Hierarchies.Condition.ParentNodeSymbol]]
+  ```
+
+  ![Example of using ParentNode symbol in rule condition](./media/hierarchy-with-parentnode-symbol-in-condition.png)
+
+- [Ruleset variables](../Advanced/RulesetVariables.md#using-variables-in-rule-condition) to  dynamically enable / disable the rule.
+
+  ```ts
+  [[include:Hierarchies.Condition.RulesetVariables.Ruleset]]
+  ```
+
+  | Ruleset variable values                                  | Result                                                                                                                           |
+  | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------- |
+  | `DISPLAY_A_NODES = false`<br />`DISPLAY_B_NODES = false` | ![Example of using ruleset variables in rule condition](./media/hierarchy-with-ruleset-variables-in-condition-none-set.png)      |
+  | `DISPLAY_A_NODES = false`<br />`DISPLAY_B_NODES = true`  | ![Example of using ruleset variables in rule condition](./media/hierarchy-with-ruleset-variables-in-condition-partially-set.png) |
+  | `DISPLAY_A_NODES = true`<br />`DISPLAY_B_NODES = true`   | ![Example of using ruleset variables in rule condition](./media/hierarchy-with-ruleset-variables-in-condition-all-set.png)       |
+
+### Attribute: `requiredSchemas`
+
+A list of ECSchema requirements that need to met for the rule to be used. See more details [here](../Advanced/SchemaRequirements.md).
+
+```ts
+[[include:Hierarchies.RequiredSchemas.Ruleset]]
+```
+
+### Attribute: `priority`
+
+Defines the order in which rules are handled - higher priority means the rule is handled first. If priorities are equal, the rules are handled in the order they're defined. The attribute may be especially useful when combined with [`onlyIfNotHandled` attribute](#attribute-onlyifnothandled).
+
+```ts
+[[include:Hierarchies.Priority.Ruleset]]
 ```
+
+![Example of using priority attribute](./media/hierarchy-with-priority-attribute.png)
+
+### Attribute: `onlyIfNotHandled`
+
+Tells the library that the rule should only be handled if no other rule of the same type was handled previously (based on rule priorities and definition order). This allows adding fallback rules which can be overriden by higher-priority rules.
+
+```ts
+[[include:Hierarchies.OnlyIfNotHandled.Ruleset]]
+```
+
+![Example of using onlyIfNotHandled attribute](./media/hierarchy-with-onlyifnothandled-attribute.png)
+
+### Attribute: `stopFurtherProcessing`
+
+Stop processing rules that have lower priority. Used in cases when recursion suppression is needed.
+
+**Note:** If this flag is set, `specifications` and `subConditions` are not processed.
```

Comment:
Maybe put notes in blockquotes for style?

```suggestion
> **Note:** If this flag is set, `specifications` and `subConditions` are not processed.
```

Reply:
There are also other places in `ChildNodeRule.md` and `RootNodeRule.md` where this suggestion would apply.

Reply:
updated